akka-docs/src/main/paradox/discovery/index.md

```diff
@@ -244,7 +244,7 @@ any Akka Management module should be used if also using Akka Discovery.
 Migration steps:
 
 * Any custom discovery method should now implement `akka.discovery.ServiceDiscovery`
-* `discovery-method` now has to be a configuration location under `akka.discovery` with at minimum a property `class` specifying the fully qualified name of the implementation of `akka.discovery.ServiceDiscovery`. 
+* `discovery-method` now has to be a configuration location under `akka.discovery` with at minimum a property `class` specifying the fully qualified name of the implementation of `akka.discovery.ServiceDiscovery`
```

Comment:
Isn't this dot needed because the next sentence is also part of the second bullet? https://doc.akka.io/docs/akka/snapshot/discovery/index.html#migrating-from-akka-management-discovery

```suggestion
* `discovery-method` now has to be a configuration location under `akka.discovery` with at minimum a property `class` specifying the fully qualified name of the implementation of `akka.discovery.ServiceDiscovery`.
```

Reply:
There are three `li` and this is the only one with an ending period `.`.

I used a majority approach. Instead, all sentences should end with a `.` period.

Reply:
Oh, wait! This is a single `li` split into multiple lines. I see now. Your suggestion is correct. Thanks @raboof